frontend/libs/studio-components/src/components/StudioProperty/StudioPropertyFieldset/StudioPropertyFieldset.stories.tsx

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import type { Meta, StoryFn } from '@storybook/react';
+import { StudioProperty } from '../index';
+import { StudioTextarea } from '../../StudioTextarea';
+import { StudioButton } from '../../StudioButton';
+import { TrashIcon, FloppydiskIcon, XMarkIcon } from '@studio/icons';
+import { StudioTextfield } from '../../StudioTextfield';
+
+type Story = StoryFn<typeof StudioProperty.Fieldset>;
+
+const meta: Meta = {
+  title: 'Components/StudioProperty/Fieldset',
+  component: StudioProperty.Fieldset,
```

Comment:
It looks like it does not make sense making it possible to control `children` and `menuBar` props for this component - maybe we could add this?
```
argTypes: {
    ...,
    children: {
      table: { disable: true },
    },
    menubar: {
      table: { disable: true },
    },
  }
```